### PR TITLE
[EDU-970] - Standardize newly written tutorials

### PR DESCRIPTION
**Pull request by marklewin**
> _NOTE TO REVIEWERS_ - please **do not review PRs in the `DRAFT` state**, as the PR may change substantially before it is ready to review. Thanks.

## Description

This work is to update the newly-rewritten Publish/Subscribe Messaging App and JWT Authentication tutorials to use the same heading structure as used in the Tracking Connected Clients with Presence tutorial (defined in the [Tutorial Writing Guidelines](https://ably.atlassian.net/wiki/spaces/DEVX/pages/2261123086/Tutorial+Writing+Guidelines)).

Namely:

- Same order, casing, and imperative rather than gerund in the headings
- Remove live demo unless it exactly mirrors the tutorial project (which it does in the case of the JWT auth tutorial)

* [EDU-970](https://ably.atlassian.net/browse/EDU-970).

## Review

Instructions on how to review the PR. 

View the Publish/Subscribe Messaging App and JWT Authentication tutorials and ensure that they render correctly in web vide and that the headings match the structure of the Tracking Connected Clients with Presence tutorial.

### CODE REVIEW

**Review by tbedford — CHANGES_REQUESTED** [review]
Looks good. Couple of points. Probably should update the metadata (last_updated).

Should we make all headings consistent? 

image

**Comment by marklewin** [comment]
Well spotted @tbedford - forgot to push last commit 🤦

File: content/tutorials/publish-subscribe.textile [hunk 1]
```diff
@@ -951,50 +799,146 @@ blang[go].
...
-      log("[Received] " + msg.data);
-    });
+  bc[sh]. git checkout publish-subscribe-java
 
-    $('button#send-message').on('click', function() {
-      var text = $('input#message-text').val();
-      log("[Publishing...] " + text);
-      channel.publish('msg', text);
-    });
+  Then you can run project inside your console. Be sure to switch into project's directory and then use these commands in your terminal:
```
Comment (Sapples) [inline content/tutorials/publish-subscribe.textile:815]: intro **the** project's directory

**Review by Sapples — CHANGES_REQUESTED** [review]
Couple small comments, otherwise LGTM.

File: content/tutorials/jwt-authentication.textile [hunk 2]
```diff
@@ -15,7 +15,7 @@ excerpt: Learn how to issue Ably JWTs for your users, configure their capabiliti
 group: sdk
 index: 55
 date_published: '2018-04-25T16:49:28+02:00'
-last_updated: '2022-08-10T09:35:45+00:00'
+last_updated: '2022-09-021T09:35:45+00:00'
```
Comment (tbedford) [inline content/tutorials/jwt-authentication.textile:18]: Should this be `21` rather than `021`?

Comment (marklewin) [reply content/tutorials/jwt-authentication.textile:18]: Fixed!

File: content/tutorials/publish-subscribe.textile (hunk 1 above)
Comment (marklewin) [reply content/tutorials/publish-subscribe.textile:815]: Good spot! Thanks.

File: content/tutorials/publish-subscribe.textile [hunk 3]
```diff
@@ -951,50 +799,146 @@ blang[go].
...
+
+  Checkout the tutorial branch:
+
+  bc[sh]. git checkout publish-subscribe-python
+
+  And then run the demo locally by adding your "Ably API key":https://faqs.ably.com/setting-up-and-managing-api-keys to @server.py@, install the required libraries:
+
+  bc[sh]. pip install ably web.py
+
+  and run the web server @python server.py@.
```
[the remark this answers is not in the document]
Comment (marklewin) [reply content/tutorials/publish-subscribe.textile:923]: Nice

**Review by Sapples — APPROVED** [review]
LGTM

**Review by tbedford — APPROVED** [review]

**Pull request merged**